bundles/AdminBundle/Resources/public/js/pimcore/document/editables/select.js

```diff
@@ -22,10 +22,23 @@ pimcore.document.editables.select = Class.create(pimcore.document.editable, {
 
         config.listeners = {};
 
+        // onchange event
+        if (config.onchange) {
+            config.listeners.select = eval(config.onchange);
+        }
+
+        if (config["required"]) {
+            config["reload"] = true;
```

Comment:
why we need to set reload to true, if required?

Reply:
We implemented that to make sure the select is revalidated on change. If you don't like it, I will remove it ofc.

Reply:
yes, please remove this check and just use on select listener `this.element.on("select", this.checkValue.bind(this, true));` to revalidate changes in render method.